cirq/ops/pauli_gates.py

```diff
@@ -15,7 +15,7 @@
 from typing import Union, TYPE_CHECKING, Tuple, cast
 
 from cirq import value
-from cirq.ops import common_gates, raw_types
+from cirq.ops import common_gates, raw_types, identity as identity
```

Comment:
Here too.

Reply:
@timothyman This hasn't been resolved yet because ` identity as identity` is still here.

Reply:
@vtomole Are you sure? Not on my branch, and not on the files changed here.

Reply:
I've had this happen to me where the diff doesn't change. Maybe its a caching thing. Anyway, this all looks good to me.